content/posts/2016/whats-new-in-pytest-30.md

```diff
@@ -0,0 +1,222 @@
+Title: What's new in pytest 3.0
+Date: 2016-07-12 12:00
+Tags: whatsnew
+Authors: nicoddemus
+Status: draft
+Summary: Summary of new the features in pytest 3.0, compared to 2.9. 
+
+This blog post provides a short overview of some of the major features and changes in pytest 3.0. See the [CHANGELOG](http://doc.pytest.org/en/latest/changelog.html) for the complete list.
+
+# New entry point: ``pytest``
```

Comment:
I find "entry point" a rather funny name as it reminds me too much of setuptools entrypoints.  Maybe something like "Additional command alias".  That also emphasises more that the "py.test" is not going anywhere (which I obviously like ;-)).